lua/nvim_lsp/dartls.lua

```diff
@@ -11,12 +11,10 @@ local find_dart_sdk_root_path = function()
   elseif vim.fn["executable"]("flutter") == 1 then
     local flutter_path = vim.fn["resolve"](vim.fn["exepath"]("flutter"))
     local flutter_bin = vim.fn["fnamemodify"](flutter_path, ":h")
-    local dart_sdk_root_path = flutter_bin.."/cache/dart-sdk/bin/dart"
-    if vim.fn["executable"](dart_sdk_root_path) == 1 then
-      return dart_sdk_root_path
-    end
+    return flutter_bin.."/cache/dart-sdk/bin/dart"
+  else
+    return ''
```

Comment:
would be better to return some reasonable default here, or at least `unknown`. Empty strings are harder to debug and result in strange error messages.

Reply:
In the case of Dart language server, if we cannot get here, we cannot use it without fail. Here, I will return a null character, and I will implement healthcheck displaying a warning if this is a null character.
What do you think?

Reply:
You mean nil? Nul char would be weird, let's avoid surprises.

Reply:
Oh sorry I mean ''.

Reply:
This commit shows I want to say. df4087fa4d175582e5be6317a9165fb834de63c5

Reply:
That commit looks like something else. Just return `unknown`, or empty string, or `nil`. Let's not overcomplicate it.